app/packs/src/components/textTemplateToolbar/ToolbarTemplateCreator.js

```diff
@@ -172,7 +171,6 @@ export default class ToolbarTemplateCreator extends React.Component {
             />
             <Select
               className="me-2 col-10 f-5"
-              ref={selectRef.ref}
```

Comment:
I think we need those refs. They are used to pull the values out of the components when saving. I have thought about rewriting the whole state management in this component but decided against it, because it seemed like too much work.

Reply:
I removed this refs because they throw warnings
`Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?`
But I haven't tried to save the toolbar.
This doesn't work anymore (with or without the refs)

So I will look for an other solution now